aptos-move/aptos-release-builder/src/lib.rs

```diff
@@ -7,7 +7,8 @@ pub mod validate;
 pub use components::{ReleaseConfig, ReleaseEntry};
 use once_cell::sync::Lazy;
 
-const RELEASE_CONFIG: &str = include_str!("../data/release.yaml");
+// Update me after branch cut.
+const RELEASE_CONFIG: &str = include_str!("../data/release-v1.4.yaml");
```

Comment:
this is not great that we need to update the file name after the branch cut. maybe, what we can do is still use release.yaml as the latest release file, and move the old stuff to release-v1.x.yaml for tracking purpose for now?

Reply:
To second this, its much easier to automate clearing and rotating the old version (changing yaml files)

Updating a const in Rust file is pretty fragile and would be less certain / automatable than rotating a yaml file

Thinking on branch cut, we would just need to rotate / clear the yaml on main, instead of changing the rust as well